theatre/core/src/propTypes/index.ts

```diff
@@ -17,6 +17,12 @@ import type {
 import {sanitizeCompoundProps} from './internals'
 import {propTypeSymbol} from './internals'
 
+// Notes on naming:
+// As of now, prop types are either `simple` or `composite`.
+// The compound type is a composite type. So is the upcoming enum type.
+// Composite types are not directly sequenceable yet. Their simple sub/ancestor props are.
+// We’ll provide a nice UX to manage keyframing of multiple sub-props.
+
 const validateCommonOpts = (fnCallSignature: string, opts?: CommonOpts) => {
```

Comment:
Added a comment in 64874d644b439739a87b2b2969a63ad5d63fa2e5 that says "See references for examples". Since `validateCommonOpts()` is not used outside this module and its references are accessible, I thought it's easy to see examples of `fnCallSignature`.

For example: https://github.com/theatre-js/theatre/blob/64874d644b439739a87b2b2969a63ad5d63fa2e5/theatre/core/src/propTypes/index.ts#L192